backend/packages/Upgrade/test/utils/database.ts

```diff
@@ -19,21 +22,42 @@ export const createDatabaseConnection = async (): Promise<Connection> => {
     password: env.db.password,
     logging: env.db.logging as LoggerOptions,
     entities: env.app.dirs.entities,
-    migrations: env.app.dirs.migrations,
-  });
-  return connection;
+    migrations: migrations,
```

Comment:
I successfully ran tests using `env.app.dirs.migration` instead of importing the variable and it worked fine.

Reply:
Lets go with @bcb37 suggestion till they force us to upgrade :)